src/cirrus/docs/components/workflows/batch.rst

```diff
@@ -1,10 +1,10 @@
 Batch tasks in workflows
```

Comment:
A large section that would be great to have is Bring Your Own Container (BYOC) - "So you have a docker container, how do you modify it to work with cirrus?"

I think we've accepted that this is not going to be a first-class citizen since it varies wildly, but general guidance here would be very helpful, internally and externally.

Reply:
this is a good point. some of this is described in the `Docker Image` section in Tasks, but I don't know if we define anywhere what the task.py should look like

Reply:
added